lib/openid_connect/endpoints/endpoint_start_point.rb

```diff
@@ -0,0 +1,34 @@
+module OpenidConnect
+  module Endpoints
+    class EndpointStartPoint < Endpoint
+      def initialize(current_user)
+        super(current_user)
+      end
+
+      def handle_response_type(req, res)
```

Comment:
Unused method argument - `res`. If it's necessary, use `_` or `_res` as an argument name to indicate that it won't be used.